restock-shop.lic

```diff
@@ -12,98 +12,95 @@ class ShopRestock
   include DRCT
 
   def initialize
-    settings = get_settings
-    ensure_copper_on_hand(40_000, settings)
-
-    @shop_id = 7943
-    @inner_id = 9099
```

Comment:
💡 Have you considered updating Cidemon's YAML with the data removed from here?

Reply:
I can update his yaml with his shop restock items, but his shop is gone so I'll leave the id's and entrance type blank

Reply:
👍

Reply:
Done